src/shaders/multiTextureFragment.glsl

```diff
@@ -2,9 +2,115 @@ precision highp float;
 
 uniform sampler2D uSampler0;
 uniform sampler2D uSampler1;
+uniform sampler2D uSampler2;
 
 varying highp vec2 vUv;
 
+vec3 mod289(vec3 x)
```

Comment:
https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode

pasale un reformat a este copypaste jajajajaj